docs/schema-v2.md

```diff
@@ -206,213 +608,40 @@ More information on custom blend [here](blend.md).
 }
 ```
 
-### Conditions Object
-
-Specifies when and where a skybox should render. All fields are optional.
-
-**Specification**
-
-|    Name   |         Datatype        |                                  Description                                  |             Default value            |
-|:---------:|:-----------------------:|:-----------------------------------------------------------------------------:|:------------------------------------:|
-| `biomes`  | Array of Namespaced Ids | Specifies a list of biomes that the skybox should be rendered in              |       Empty Array (all biomes)       |
-| `worlds`  | Array of Namespaced Ids | Specifies a list of worlds that the skybox should be rendered in              |       Empty Array (all worlds)       |
-| `weather` | Array of Weathers       | Specifies a list of weather conditions that the skybox should be rendered in  | Empty Array (all weather conditions) |
-| `xRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all x coordinates)    |
-| `yRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all y coordinates)    |
-| `zRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all z coordinates)    |
-
-
-**Example**
+# Full Example
+Here is a full skybox file from [Awesome Skies](https://curseforge.com/minecraft/texture-packs/awesome-skies), courtesy of [@heymanMC](https://github.com/heymanMC):
```

Comment:
I don't think this is the way to go about this.  Really we should have an example of our own instead of using one from Awesome Skies.  I can write up a quick one tonight.